src/components/image/index.js

```diff
@@ -0,0 +1,67 @@
+import React from "react"
+import PropTypes from "prop-types"
+import Img from "gatsby-image"
+import classNames from "classnames/bind"
+import dig from "object-dig"
+import styles from "./styles.module.scss"
+
+const Image = ({ alt, background, className, src, width, ...props }) => {
+  const classes = classNames(styles.image, {
+    [styles.has_background]: background,
+    [className]: className
+  })
+
+  // ---------------------------------------- | Gastby Image
+
+  if (dig(src, "childImageSharp", "fluid") || dig(src, "childImageSharp", "fixed")) {
+    return (
+      <div className={classes}>
```

Comment:
I don't think we should wrap these images in divs. We did this for the GAIG project, and we can implement that as necessary in other projects. But here I think we're okay to just render the image directly.